server/svix-server/src/core/otel_spans.rs

```diff
@@ -128,7 +128,7 @@ impl<B> MakeSpan<B> for AxumOtelSpanCreator {
         );
 
         if let Some(key) = idempotency_key {
-            span.record("idempotency_key", &key);
+            span.record("idempotency_key", key);
```

Comment:
Why all of these unrelated changes?

Reply:
I was getting clippy failures in the build https://github.com/svix/svix-webhooks/actions/runs/3136424848/jobs/5093319099#step:8:727

I can remove these diffs, but I didn't think the PR would be mergable unless they passed

Reply:
Fixed